bytestring/common/src/ByteString.kt

```diff
@@ -37,6 +38,23 @@ public fun ByteString(vararg bytes: Byte): ByteString = if (bytes.isEmpty()) {
     ByteString.wrap(bytes)
 }
 
+/**
+ * Wraps given [bytes] into a byte string.
+ *
```

Comment:
Is it worth clarifying (a seemingly obvious thing) how the written bytes will be read from ByteString (as `.toByte()`)?

Reply:
It definitely is